src/test/compile-fail/issue-28472.rs

```diff
@@ -13,10 +13,10 @@
 extern {
   fn foo();
 
-  pub //~ ERROR a value named `foo` has already been defined
+  pub //~ ERROR E0428
```

Comment:
Could you keep using messages for testing and avoid error codes outside of `E0XXX.rs` tests.